setup.py

```diff
@@ -12,7 +12,10 @@
 algorithms in a package-independent manner.
 """
 
-import sys
+from __future__ import with_statement
+from __future__ import absolute_import
+from setuptools import setup, find_packages
+from io import open  # pylint:disable=redefined-builtin
```

Comment:
I don't think this is correct. Wouldn't this import `cclib.io`? Nothing should need to be imported to read in a plain text file.

Reply:
No, this is to get a consistent open function in python 2.7 and 3.x.
AFAIK the import manager does not recurse through directories to find a module matching the name to import and your io module is several layers deeper than the setup.py script.

http://python-future.org/compatible_idioms.html
```
# Python 2 and 3: alternative 1
from io import open
f = open('myfile.txt', 'rb')
data = f.read()              # as bytes
text = data.decode('utf-8')  # unicode, not bytes
```